pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "exasol-ai-lab"
-version = "3.0.0"
+version = "4.0.0"
```

Comment:
```suggestion
version = "3.1.0"
```

Reply:
I wasn't sure what number to put. It looked like when we make a release we increment the major version number. I don't mind either way.

Reply:
the last two version had breaking changes, this one is only an internal change and I will add some bugfixes soon, that means we have soon a 3.1.0